ehealthid-cli/src/main/java/com/oviva/ehealthid/cli/MTlsRefreshCommand.java

```diff
@@ -0,0 +1,130 @@
+package com.oviva.ehealthid.cli;
+
+import com.nimbusds.jose.JOSEException;
+import com.nimbusds.jose.jwk.*;
+import com.nimbusds.jose.jwk.gen.ECKeyGenerator;
+import com.nimbusds.jose.util.Base64;
+import com.nimbusds.oauth2.sdk.id.Issuer;
+import com.nimbusds.oauth2.sdk.util.X509CertificateUtils;
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.security.cert.CertificateEncodingException;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Date;
+import java.util.List;
+import java.util.concurrent.Callable;
+import org.bouncycastle.operator.OperatorCreationException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
+
+@CommandLine.Command(
+    name = "tls-refresh",
+    mixinStandardHelpOptions = true,
+    description = "Generate or Refresh the mTLS certificates in the Signing Key.")
+public class MTlsRefreshCommand implements Callable<Integer> {
+
+  @CommandLine.Option(
+      names = {"-i", "--iss", "--issuer-uri"},
+      description = "the issuer uri of the 'Fachdienst' identiy provider",
+      required = false)
```

Comment:
```suggestion
      required = true)
```
No? Below you at least assume its present :)

Reply:
Yes, that was from a previous iteration.